src/schemes/boundary/dummy_particles/dummy_particles.jl

```diff
@@ -378,83 +417,128 @@ function compute_pressure!(boundary_model, ::Union{PressureMirroring, PressureZe
+                                 distance, viscosity, cache, pressure, pressure_offset)
     end
 end
 
-@inline function adami_pressure_inner!(boundary_model, system, neighbor_system,
-                                       v_neighbor_system, particle, neighbor, pos_diff,
-                                       distance, viscosity, cache, pressure)
+@inline function boundary_pressure_inner!(boundary_model, boundary_density_calculator,
+                                          system, neighbor_system::FluidSystem, v,
+                                          v_neighbor_system, particle, neighbor, pos_diff,
+                                          distance, viscosity, cache, pressure,
+                                          pressure_offset)
     density_neighbor = particle_density(v_neighbor_system, neighbor_system, neighbor)
 
-    resulting_acc = neighbor_system.acceleration -
-                    current_acceleration(system, particle)
+    resulting_acceleration = neighbor_system.acceleration -
+                             current_acceleration(system, particle)
 
     kernel_weight = smoothing_kernel(boundary_model, distance)
 
-    pressure[particle] += (particle_pressure(v_neighbor_system, neighbor_system,
-                                             neighbor) +
-                           dot(resulting_acc, density_neighbor * pos_diff)) *
-                          kernel_weight
+    pressure[particle] += (pressure_offset
+                           +
+                           particle_pressure(v_neighbor_system, neighbor_system,
+                                             neighbor)
+                           +
+                           dynamic_pressure(boundary_density_calculator, density_neighbor,
+                                            v, v_neighbor_system, pos_diff, distance,
+                                            particle, neighbor, system, neighbor_system)
+                           +
+                           dot(resulting_acceleration,
+                               density_neighbor * pos_diff)) * kernel_weight
 
     cache.volume[particle] += kernel_weight
 
     compute_smoothed_velocity!(cache, viscosity, neighbor_system, v_neighbor_system,
                                kernel_weight, particle, neighbor)
 end
 
+@inline function dynamic_pressure(boundary_density_calculator, density_neighbor, v,
+                                  v_neighbor_system, pos_diff, distance, particle, neighbor,
+                                  system,
+                                  neighbor_system)
+    return zero(density_neighbor)
+end
+
+@inline function dynamic_pressure(boundary_density_calculator::BernoulliPressureExtrapolation,
+                                  density_neighbor, v, v_neighbor_system, pos_diff,
+                                  distance, particle, neighbor, system::BoundarySystem,
+                                  neighbor_system)
```

Comment:
```suggestion
                                  distance, particle, neighbor, 
                                  system::BoundarySystem, neighbor_system)
```

Reply:
?

Reply:
IMHO, it's more readable when the systems are in one line. I don't think we have a convention.